pkg/component/labels/labels.go

```diff
@@ -5,30 +5,43 @@ import (
 	"github.com/redhat-developer/odo/pkg/util"
 )
 
-// ComponentLabel is a label key used to identify the component name
-const ComponentLabel = "app.kubernetes.io/instance"
+// KubernetesInstanceLabel is a label key used to identify the component name
+const KubernetesInstanceLabel = "app.kubernetes.io/instance"
 
-// ComponentTypeLabel is Kubernetes label that identifies the type of a component being used
-const ComponentTypeLabel = "app.kubernetes.io/name"
+// KubernetesNameLabel is Kubernetes label that identifies the type of a component being used
+const KubernetesNameLabel = "app.kubernetes.io/name"
 
-const ComponentTypeAnnotation = "odo.dev/project-type"
+// KubernetesManagedByLabel ...
+const KubernetesManagedByLabel = "app.kubernetes.io/managed-by"
 
-// ComponentDeployLabel ...
-const ComponentDeployLabel = "Deploy"
+// ComponentUnknownLabel is the label that is used to display something we do not know
+const ComponentUnknownLabel = "<unknown>"
 
-// ComponentModeLabel ...
-const ComponentModeLabel = "odo.dev/mode"
+// ComponentDevName ...
+const ComponentDevName = "Dev"
 
-// ComponentProjectTypeLabel ...
-const ComponentProjectTypeLabel = "odo.dev/project-type"
+// ComponentDeployName ...
+const ComponentDeployName = "Deploy"
+
+// ComponentNoneName ...
+const ComponentNoneName = "None"
+
+// ComponentPushedName ...
+const ComponentPushedName = "Pushed"
```

Comment:
This is not used as a label or its value it should not be in this file.

Similar constant is already defined in [pkg/component/types.go](https://github.com/redhat-developer/odo/pull/5551/files#diff-fe93776d90006632b5ac56768583c90eb9621e641f95ffa44d86cb688d08e208)